src/EditorFeatures/Core.Wpf/Adornments/AdornmentManager.cs

```diff
@@ -22,42 +22,26 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.Adornments
     /// <summary>
     /// UI manager for graphic overlay tags. These tags will simply paint something related to the text.
     /// </summary>
-    internal class AdornmentManager<T> where T : GraphicsTag
+    internal abstract class AdornmentManager<T> where T : GraphicsTag
     {
-        private readonly object _invalidatedSpansLock = new object();
+        protected readonly object _invalidatedSpansLock = new object();
```

Comment:
very wary about exposing a lock.

Reply:
was not used by the subclasses, made it private readonly